src/components/experiments/single-view/results/CondensedLatestAnalyses.tsx

```diff
@@ -85,7 +85,7 @@ export default function CondensedLatestAnalyses({
         .map(
           (analyses) =>
             //  istanbul ignore next; We don't need to test empty analyses as we filter out all undefined values
-            _.last(analyses)?.recommendation,
+            _.last(analyses)?.recommendation.chosenVariationId,
         )
         .filter((recommendation) => !!recommendation)
       const uniqueRecommendations = _.uniq(recommendations.map((recommendation) => JSON.stringify(recommendation)))
```

Comment:
In addition to the naming, it looks like the `filter()` call won't work as expected. How about we change it to something like this?

```suggestion
            _.last(analyses)?.recommendation,
        )
        .filter((recommendation) => !!recommendation)
      const recommendationConflict = _.uniq(recommendations.map((recommendation) => recommendation.chosenVariationId)) > 1
```

That is, leave the `recommendations` array as it is, and change the `recommendationConflict` variable.

Reply:
I don't see why filter won't work?
Also before this PR goes out `chosenVariationId` will be optional chained, in case that is something you are missing?

Reply:
Ahh I didn't realise `chosenVariationId` set to `null` signified something.